app/javascript/utils/fieldSpecs.js

```diff
@@ -2,9 +2,15 @@ import { each, replace, get, toLower, includes, isString, camelCase, startCase,
 import moment from 'moment'
 
 import utils from '~/utils/utils'
+import formUtils from '~/utils/formUtils'
 
 export const getFormatType = (field) => {
-  if (includes(toLower(field), 'date')) { return 'date' } else { return null }
+  if (includes(toLower(field), 'date')) {
+    return 'date'
```

Comment:
this seems a bit sketchy to me. I'm not sure what the new parking price PR used for a field name but I don't think that it has rent or income

Reply:
Yea I was following a similar helper but looking at better options

Reply:
updated to what we discussed